tests/integration_tests/functional/test_mmds.py

```diff
@@ -268,6 +270,73 @@ def test_imds_response(test_microvm_with_ssh, network_config):
                                 ' unsupported type.')
 
 
+def test_larger_than_mss_payloads(test_microvm_with_ssh, network_config):
+    """Test MMDS content for payloads larger than MSS."""
+    test_microvm = test_microvm_with_ssh
+    test_microvm.spawn()
+
+    # The MMDS is empty at this point.
+    response = test_microvm.mmds.get()
+    assert test_microvm.api_session.is_status_ok(response.status_code)
+    assert response.json() == {}
+
+    test_microvm.basic_config(vcpu_count=1)
+    _tap = test_microvm.ssh_network_config(
+        network_config,
+        '1',
+        allow_mmds_requests=True
+    )
+
+    test_microvm.start()
+
+    # Make sure MTU is 1500 bytes.
+    ssh_connection = net_tools.SSHConnection(test_microvm.ssh_config)
+    cmd = 'ip a s eth0 | grep -i mtu | tr -s " " | cut -d " " -f 4,5'
+    _, stdout, stderr = ssh_connection.execute_command(cmd)
+    _assert_out(stdout, stderr, "mtu 1500\n")
+
+    # These values are usually used by booted up guest network interfaces.
+    mtu = 1500
+    ipv4_packet_headers_len = 20
+    tcp_segment_headers_len = 20
+    mss = mtu - ipv4_packet_headers_len - tcp_segment_headers_len
```

Comment:
nit: declare constants for these

Reply:
These variables are used only inside this test. I would like to keep them visible only for the scope of this test.